dts/bindings/vendor-prefixes.txt

```diff
@@ -439,6 +439,7 @@ opalkelly	Opal Kelly Incorporated
 opencores	OpenCores.org
 openisa	open-isa.org
 openrisc	OpenRISC.io
+openthread	OpenThread-specific binding
```

Comment:
```suggestion
openthread	OpenThread
```

Reply:
I've added "-specific binding" to keep consistency with `Zephyr-specific binding`  and `Linux-specific binding`, as OpenThread is not a vendor by itself, but needs some dts bindings to work with additional configurations. But I can change it to just `OpenThread` :) 
Which do you think would be better?

Reply:
> I've added "-specific binding" to keep consistency with Zephyr-specific binding and Linux-specific binding

The more you know :)

Maybe we can use the domain `OpenThread.io`?

Reply:
`OpenThread.io` sounds good to me. Renamed